### ISSUE TITLE
Unused function prototype in aws_demo.h

### ISSUE THREAD

**Issue by jnz86** [aws/amazon-freertos#1821]
**Describe the bug**
`aws_demo.h`   line56
```
void vApplicationInit( void );
```
Function appears only once as a prototype in the whole solution and never in bootloader.

**Comment by leegeth** [comment]
@jnz86,
Thanks for bringing this issue to our attention. We will evaluate and fix this.

**Comment by leegeth** [comment]
We have fixed this issue by PR #1824. Closing this issue. Please open a new issue if there are any other questions.
